lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -62,15 +68,16 @@ def paintEvent(self, e):
             return
 
         painter = QtGui.QPainter(self)
-
-        if self._bbox_queue:
+        with self._bbox_queue_lock:
             bbox_queue = self._bbox_queue
-        else:
-            painter.eraseRect(self.rect())
-            bbox_queue = [
-                Bbox([[0, 0], [self.renderer.width, self.renderer.height]])]
+            self._bbox_queue = []
+
         self._bbox_queue = []
         for bbox in bbox_queue:
+            if bbox is None:
```

Comment:
If _any_ of them are `None` can short circuit the whole thing and just copy once.